lib/DDG/Spice/IndeedJobs.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::IndeedJobs;
+# ABSTRACT: Search for jobs on Indeed.
+# Reviews DDG_SPICE_INDEED_JOBS_PUBLISHER environment variable
+# https://ads.indeed.com/jobroll/xmlfeed
+
+use DDG::Spice;
+
+primary_example_queries "java jobs";
+secondary_example_queries "perl jobs in san francisco";
+description "Indeed jobs";
+name "Indeed Jobs";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/IndeedJobs.pm";
+topics "special_interest";
+category  "reference";
+attribution github  => ['parker', 'Parker'],
+            twitter => ['pseidel', 'Parker'],
+            github  => ['tagawa', 'Daniel Davis'],
+            twitter => ['ourmaninjapan', 'Daniel Davis'],
+            web     => ['http://daniemon.com/', 'Daniel Davis'];
+
+triggers any => "job", "jobs";
+
+spice to => 'http://api.indeed.com/ads/apisearch?publisher={{ENV{DDG_SPICE_INDEED_APIKEY}}}&v=2&useragent=DuckDuckGo&userip=1.2.3.4&q=$1&l=$2&co=$3&format=json&callback={{callback}}';
+spice from => '([^/]+)/(.*?)/([^/]*)';
```

Comment:
@tagawa it's best to put optional checks at the end and only return them if they're defined otherwise you'll end up with a double slash `//` which gets automatically changed to `/` by NGINX.

```perl
spice from => '([^/]+)/([^/]+)/?([^/]+)?';
```